controllers/post.js

```diff
@@ -12,7 +19,7 @@ const getAllPosts = async (req, res) => {
 const saveNewPost = async (req, res) => {
   try {
     const post = new Post({
-      message: req.body.message,
+      content: req.body.message,
```

Comment:
```suggestion
      content: req.body.content,
```

I think you should change it as well